package.json

```diff
@@ -336,8 +336,8 @@
     "test": "node ./out/test/runTest.js",
     "test:unit": "jest",
     "test:syntax": "npm run test:grammar && npm run test:snap",
-    "test:grammar": "npx vscode-tmgrammar-test -s scope.terraform -g syntaxes/terraform.tmGrammar.json -t tests/unit/**/*.tf",
-    "test:snap": "npx vscode-tmgrammar-snap -s scope.terraform -g syntaxes/terraform.tmGrammar.json -t tests/snapshot/**/*.tf",
+    "test:grammar": "npx vscode-tmgrammar-test -s scope.terraform -g syntaxes/terraform.tmGrammar.json -t 'tests/unit/**/*.tf'",
```

Comment:
Since we're here, would it be better to use `\"` instead as we'll have to eventually do `{tf|tfvars}`, etc? Or wait until the need arises?

Reply:
I think there is no need to use `\"` here. Multiple file extensions should work with single quotes, too: `'tests/snapshot/**/*.+(tf|tfvars)'`.